website/docs/r/compute_disk.html.markdown

```diff
@@ -1,102 +1,231 @@
+  (Optional)
+  Specifies a 256-bit customer-supplied encryption key, encoded in
+  RFC 4648 base64 to either encrypt or decrypt this resource.
+* `sha256` -
+  The RFC 4648 base64 encoded SHA-256 hash of the customer-supplied
+  encryption key that protects this resource.
+  
+  
+The `source_image_encryption_key` block supports:
+* `raw_key` -
+  (Optional)
+  Specifies a 256-bit customer-supplied encryption key, encoded in
+  RFC 4648 base64 to either encrypt or decrypt this resource.
+* `sha256` -
+  The RFC 4648 base64 encoded SHA-256 hash of the customer-supplied
+  encryption key that protects this resource.
+  
+  
+The `source_snapshot_encryption_key` block supports:
+* `raw_key` -
+  (Optional)
+  Specifies a 256-bit customer-supplied encryption key, encoded in
+  RFC 4648 base64 to either encrypt or decrypt this resource.
+* `sha256` -
+  The RFC 4648 base64 encoded SHA-256 hash of the customer-supplied
+  encryption key that protects this resource.
+  
+  
+
+* (Deprecated) `disk_encryption_key_raw`:  This is an alias for
+  `disk_encryption_key.raw_key`.  It is deprecated to enhance
+  consistency with `source_image_encryption_key` and
+  `source_snapshot_encryption_key`.
 ## Attributes Reference
 
-In addition to the arguments listed above, the following computed attributes are
-exported:
-
-* `disk_encryption_key_sha256` - The [RFC 4648 base64]
-    (https://tools.ietf.org/html/rfc4648#section-4) encoded SHA-256 hash of the
-    [customer-supplied encryption key](https://cloud.google.com/compute/docs/disks/customer-supplied-encryption)
-    that protects this resource.
-
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `creation_timestamp` -
+  Creation timestamp in RFC3339 text format.
+* `last_attach_timestamp` -
+  Last attach timestamp in RFC3339 text format.
+* `last_detach_timestamp` -
+  Last dettach timestamp in RFC3339 text format.
+* `users` -
+  Links to the users of the disk (attached instances) in form:
+  project/zones/zone/instances/instance
+* `source_image_id` -
+  The ID value of the image used to create this disk. This value
+  identifies the exact image that was used to create this persistent
+  disk. For example, if you created the persistent disk from an image
+  that was later deleted and recreated under the same name, the source
+  image ID would identify the exact version of the image that was used.
+* `source_snapshot_id` -
+  The unique ID of the snapshot used to create this disk. This value
+  identifies the exact snapshot that was used to create this persistent
+  disk. For example, if you created the persistent disk from a snapshot
+  that was later deleted and recreated under the same name, the source
+  snapshot ID would identify the exact version of the snapshot that was
+  used.
 * `self_link` - The URI of the created resource.
 
-* `users` - The Users of the created resource.
-
-* `label_fingerprint` - The fingerprint of the assigned labels.
```

Comment:
label_fingerprint should be added to the docs too

Reply:
Done.